app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -85,46 +69,56 @@ To be eligible for a scholarship, you’ll need at least one of the following:
-
-You may also be eligible if you do an opt-in QTS course in secondary computing, languages, mathematics or physics. This is where you start a standard undergraduate degree but then opt-in to initial teacher training to get QTS.
+| Subject                                                               | Bursary  | Scholarship      |
+|-----------------------------------------------------------------------| -------: | ---------------: |
+| **Art and design**                                                    | $bursaries_postgraduate_artanddesign$  |                  |
+| **Biology**                                                           | $bursaries_postgraduate_biology$  |                  |
+| **Chemistry**                                                         | $bursaries_postgraduate_chemistry$  | $scholarships_chemistry$          |
+| **Computing**                                                         | $bursaries_postgraduate_computing$  | $scholarships_computing$          |
+| **Design and technology**                                             | $bursaries_postgraduate_designandtechnology$  |                  |
+| **English**                                                           | $bursaries_postgraduate_english$  |                  |
+| **Geography**                                                         | $bursaries_postgraduate_geography$  |                  |
+| **Languages**<br/> (French, German and Spanish only)                  | $bursaries_postgraduate_languagesfrenchgermanspanish$  | $scholarships_languagesfrenchgermanspanish$      |
+| **Languages**<br/> (all other languages, including ancient languages) | $bursaries_postgraduate_languagesother$  |                  |
+| **Maths**                                                             | $bursaries_postgraduate_maths$  | $scholarships_maths$         |
+| **Music**                                                             | $bursaries_postgraduate_music$  |                  |
+| **Physics**                                                           | $bursaries_postgraduate_physics$  | $scholarships_physics$        |
+| **Religious education**                                               | $bursaries_postgraduate_religiouseducation$  |                  |
 
-The amount you receive depends on the year that you start your QTS course. You do not get a bursary in every year of an undergraduate course. Depending on the type of course you do, you may get the bursary in your final year or final two years.
+Table caption: Available postgraduate bursaries and scholarships
 
-### Undergraduate bursary if you're former military personnel
+## If you think you're not eligible
```

Comment:
```suggestion
## If you're not eligible for a bursary or scholarship
```
Not sure `think` is needed here?

Also - should make the heading clearer for screen reader users